src/main/java/fr/moribus/imageonmap/commands/maptool/DeleteOtherCommand.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2013 Moribus
+ * Copyright (C) 2015 ProkopyL <prokopylmc@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package fr.moribus.imageonmap.commands.maptool;
+
+import fr.moribus.imageonmap.commands.IoMCommand;
+import fr.moribus.imageonmap.map.ImageMap;
+import fr.moribus.imageonmap.map.MapManager;
+import fr.moribus.imageonmap.map.MapManagerException;
+import fr.zcraft.zlib.components.commands.CommandException;
+import fr.zcraft.zlib.components.commands.CommandInfo;
+import fr.zcraft.zlib.components.commands.WithFlags;
+import fr.zcraft.zlib.components.i18n.I;
+import fr.zcraft.zlib.tools.PluginLogger;
+
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Player;
+
+import java.util.List;
+import java.util.UUID;
+
+@CommandInfo (name =  "deleteother", usageParameters = "<player name> <map name>")
+@WithFlags ({"confirm"})
+public class DeleteOtherCommand extends IoMCommand
```

Comment:
The same command should be used, instead of another set for moderators. This being said, it poses the problem regarding multi-words map names.

Reply:
If we did make delete work for other players, shouldn't we also concatenate all of the commands to one? I feel it makes a little more sense as it is, and you're right- multi-word map names would pretty easily break it. (Especially if they had a playername in one)